packages/cursorless-vscode/src/ide/vscode/hats/VscodeHatRenderer.ts

```diff
@@ -132,10 +135,13 @@ export default class VscodeHatRenderer {
     if (hatsDir) {
       await this.updateShapeOverrides(hatsDir);
 
-      if (fs.existsSync(hatsDir)) {
+      try {
+        await fs.access(hatsDir);
         this.hatsDirWatcherDisposable = watchDir(hatsDir, () =>
           this.updateShapeOverrides(hatsDir),
         );
+      } catch (e) {
+        console.error("cannot watch hatsDir " + hatsDir);
```

Comment:
If we're going to log, we might as well log the exception too.
```suggestion
        console.error("cannot watch hatsDir", hatsDir, e);
```